<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ã“pticaJireh</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons/Icons.qrc">
    <normaloff>:/Icons/internet-news-reader.png</normaloff>:/Icons/internet-news-reader.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="test"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar_modulos">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProducto"/>
   <addaction name="actionUsuario"/>
   <addaction name="actionCliente"/>
   <addaction name="actionEmpresas"/>
   <addaction name="actionTiendas"/>
   <addaction name="actionAlmacen"/>
   <addaction name="actionProveedores"/>
   <addaction name="actionCompras"/>
   <addaction name="actionReportes"/>
   <addaction name="actionVentas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QToolBar" name="options_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionBuscar"/>
   <addaction name="actionNuevo"/>
   <addaction name="actionEditar"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionCancelar"/>
  </widget>
  <action name="actionProducto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/inventory-maintenance.png</normaloff>:/Icons/inventory-maintenance.png</iconset>
   </property>
   <property name="text">
    <string>Producto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUsuario">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/users.png</normaloff>:/Icons/users.png</iconset>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCliente">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/clients.png</normaloff>:/Icons/clients.png</iconset>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1349716425_doc_edit.png</normaloff>:/Icons/1349716425_doc_edit.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="toolTip">
    <string>Editar registro actual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1349716386_monotone_add_plus_round.png</normaloff>:/Icons/1349716386_monotone_add_plus_round.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="toolTip">
    <string>Nuevo Registro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1349716430_zoom.png</normaloff>:/Icons/1349716430_zoom.png</iconset>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionTiendas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1354299865_rssshop.png</normaloff>:/Icons/1354299865_rssshop.png</iconset>
   </property>
   <property name="text">
    <string>Tiendas</string>
   </property>
   <property name="toolTip">
    <string>Tienda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionEmpresas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/photo.png</normaloff>:/Icons/photo.png</iconset>
   </property>
   <property name="text">
    <string>Empresas</string>
   </property>
   <property name="toolTip">
    <string>Empresas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAlmacen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/file-roller.png</normaloff>:/Icons/file-roller.png</iconset>
   </property>
   <property name="text">
    <string>Almacenes</string>
   </property>
  </action>
  <action name="actionProveedores">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/autoship.png</normaloff>:/Icons/autoship.png</iconset>
   </property>
   <property name="text">
    <string>Proveedores</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCompras">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/shopping-cart_basic_blue.png</normaloff>:/Icons/shopping-cart_basic_blue.png</iconset>
   </property>
   <property name="text">
    <string>Compras</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1349716356_save.png</normaloff>:/Icons/1349716356_save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="toolTip">
    <string>guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCancelar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/1349716433_doc_delete.png</normaloff>:/Icons/1349716433_doc_delete.png</iconset>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionReportes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/abiword.png</normaloff>:/Icons/abiword.png</iconset>
   </property>
   <property name="text">
    <string>Reportes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionVentas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Icons.qrc">
     <normaloff>:/Icons/ticket.png</normaloff>:/Icons/ticket.png</iconset>
   </property>
   <property name="text">
    <string>Ventas</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
